### DISCUSSION TITLE
how we can do paralel plotting on a ubuntu machin?

### DISCUSSION THREAD

**Discussion by salimbasar** [Chia-Network/chia-blockchain#5031] [Support]
how we can do paralel plotting on a ubuntu machine? do you have any solution?

**Comment by cccat6** [comment]
Use CLI and write a simple script to control everything.
If you don't know how to do it. There are also many open-source scripts even a full manager you can use.

**Reply by ksdomino** [reply]
I've looked all over the place for this, does anyone know the name of the open source scripts or full manager?

**Reply by cccat6** [reply]
Search for "plotman"

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
